### PR TITLE
Updated the Tensorboards Operator Readme

### PR DESCRIPTION
**Pull request by NohaIhab**
documentation update on Usage and examples included

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,79 @@
...
-To install Kubeflow Tensorboards, run:
+## Usage
 
-    juju deploy kubeflow-tensorboards
+To get started, install Kubeflow lite bundle (See [Quick start guide](https://charmed-kubeflow.io/docs/quickstart)).
 
+Then, deploy Tensorboard Controller and Tensorboard Web App with
+```bash
+juju deploy tesnorboard-controller --trust
+juju deploy tensorboards-web-app --trust
```
Comment (DomFleischmann) [inline README.md:18]: is `--trust` required for these charms?

Comment (DomFleischmann) [reply README.md:18]: I also believe we should point to `--channel=latest/edge` here, this is the README for main so with latest/edge we actually deploy what currently on main.

File: README.md [hunk 2]
```diff
@@ -1,17 +1,79 @@
...
+```
+And Create a relation to istio pilot with
+```bash
+juju relate istio-pilot:ingress tensorboards-web-app:ingress
```
Comment (DomFleischmann) [inline README.md:22]: We are also missing the gateway relation between tensorboard-controller and istio-pilot which is added in `latest/edge`

File: README.md [hunk 3]
```diff
@@ -1,17 +1,79 @@
...
+### Key features
+* Centralised, browser-based data science workspaces: **familiar experience**
+* Multi user: **one environment for your whole data science team**
+* NVIDIA GPU support: **accelerate deep learning model training**
+* Apache Spark integration: **empower big data driven model training**
+* Ideation to production: **automate model training & deployment**
+* AutoML: **hyperparameter tuning, architecture search**
+* Composable: **edge deployment configurations available**
+
+### What’s included in Charmed Kubeflow 1.4
```
Comment (DomFleischmann) [inline README.md:48]: lets remove the 1.4 part here.

**Review by DomFleischmann — CHANGES_REQUESTED** [review]
Looks good, just some small changes required.

File: README.md [hunk 4]
```diff
@@ -1,17 +1,80 @@
...
 
-To install Kubeflow Tensorboards, run:
+## Usage
 
-    juju deploy kubeflow-tensorboards
+To get started, install Kubeflow lite bundle (See [Quick start guide](https://charmed-kubeflow.io/docs/quickstart)).
 
+Then, deploy Tensorboard Controller and Tensorboard Web App with
+```bash
+juju deploy tesnorboard-controller --channel=latest/edge
```
Comment (natalian98) [inline README.md:17]: just a small typo
```suggestion
juju deploy tensorboard-controller --channel=latest/edge
```

File: README.md [hunk 5]
```diff
@@ -1,17 +1,80 @@
...
+
+### Documentation
+Please see the [official docs site](https://charmed-kubeflow.io/docs) for complete documentation of the Charmed Kubeflow distribution.
+
+### Bugs and feature requests
+If you find a bug in our operator or want to request a specific feature, please file a bug here:
+[https://github.com/canonical/dex-auth-operator/issues](https://github.com/canonical/dex-auth-operator/issues)
+
+### License
+Charmed Kubeflow is free software, distributed under the [Apache Software License, version 2.0](https://github.com/canonical/dex-auth-operator/blob/master/LICENSE).
```
Comment (natalian98) [inline README.md:74]: How about pointing to bundle-kubeflow's [license](https://github.com/canonical/bundle-kubeflow/blob/main/LICENSE)?

File: README.md [hunk 6]
```diff
@@ -1,17 +1,80 @@
...
+* Optional 24/7 support available, [contact us here](https://charmed-kubeflow.io/contact-us) for more information
+* Optional dedicated fully managed service available, [contact us here](https://charmed-kubeflow.io/contact-us) for more information or [learn more about Canonical’s Managed Apps service](https://ubuntu.com/managed/apps).
+* Portability: Charmed Kubeflow can be deployed on any conformant Kubernetes, on any cloud or on-premise
+
+### Documentation
+Please see the [official docs site](https://charmed-kubeflow.io/docs) for complete documentation of the Charmed Kubeflow distribution.
+
+### Bugs and feature requests
+If you find a bug in our operator or want to request a specific feature, please file a bug here:
+[https://github.com/canonical/dex-auth-operator/issues](https://github.com/canonical/dex-auth-operator/issues)
```
Comment (natalian98) [inline README.md:71]: Let's point to tensorboards repo instead
```suggestion
https://github.com/canonical/kubeflow-tensorboards-operator/issues
```

**Review by DomFleischmann — APPROVED** [review]

**Pull request merged**